salt/states/boto_ec2.py

```diff
@@ -1752,5 +1749,169 @@ def private_ips_absent(name, network_interface_name=None, network_interface_id=N
 
         # there were no changes since we did not attempt to remove ips
         ret['changes'] = {}
+    return ret
+
+
+def secgroup_present(
```

Comment:
I could alter `secgroup_present` to behave like `secgroup_equals` by default (thereby being more in sync with other _present states) by adding a boolean option in the arguments. This would also remove the need to have a separate `secgroup_equals` defined.